src/Search/Variants/SearchVariant.php

```diff
@@ -186,6 +186,11 @@ public static function withCommon(array $classes = [])
         // Construct new array of variants applicable to at least one class in the list
         $commonVariants = [];
         foreach ($classes as $class => $options) {
+            // BC for numerically indexed list of classes
+            if (is_numeric($class) && isset($options['class'])) {
+                $class = $options['class'];
```

Comment:
Would you think it reasonable to throw an exception or error here if `$options['class']` is unset?
Or alternatively, add a comment to indicate that it's _always_ set?

Reply:
Other pieces of code in the module rely on the `$options['class']` and they won't work without it, so I'd say it's nearly safe to assume it exists. Someone can make the decision and I'll update it.

Reply:
I can make the decision - just add a comment reflecting this info and I'll merge it now* :)

e.g.
```diff
+ // $options['class'] is always assumed to exist throughout the code base
```

*after the test suite passes